tests/vaults/VaultManager.test.ts

```diff
@@ -91,6 +91,7 @@ describe('VaultManager', () => {
     keyManager = await KeyManager.createKeyManager({
       password,
       keysPath: keysPath,
+      rootKeyPairChange: async () => {},
```

Comment:
This callback should be optional, so if you don't pass it in, it's by default set to a noop function if you don't want to check `undefined`.

Reply:
This comment applies to all the other tests where `rootKeyPairChange` is being passed a noop function.